packages/circuit-ui/components/TopNavigation/TopNavigation.mdx

```diff
@@ -17,3 +17,9 @@ The top navigation is part of the [application shell](https://developers.google.
 <Story of={Stories.WithSideNavigation} inline={false} />
 
 When used alongside the [SideNavigation](Navigation/SideNavigation) component, the top navigation displays a hamburger button on narrow viewports which can be used to toggle the side navigation.
+
+## Accessibility
+
+To improve the navigation experience, this component includes a skip navigation link that allows users to jump directly to the main content of the page. This is particularly useful for users who rely on screen readers or keyboard navigation.
+To enable this behaviour, simply pass a hash link to your main content as the `skipNavigationLink` prop and a `skipNavigationLabel` as label.
```

Comment:
It's good practice to avoid using words such as "simple", "easy", and "just" in documentation. Whether something is simple depends on the reader's context, not the writer: https://justsimply.dev/

```suggestion
To enable this behaviour, pass a hash link to your main content as the `skipNavigationLink` prop and a `skipNavigationLabel` as label.
```